test/unit/app/browser/reducers/downloadsReducerTest.js

```diff
@@ -257,4 +277,19 @@ describe('downloadsReducer', function () {
       })
     })
   })
+
+  describe('APP_SET_STATE', function () {
+    it('DOWNLOAD_DEFAULT_PATH is empty', function () {
+      changeSettingSpy.reset()
+      downloadDefaultPath = ''
+      downloadsReducer({}, {actionType: appConstants.APP_SET_STATE})
+      assert(changeSettingSpy.withArgs(settings.DOWNLOAD_DEFAULT_PATH, `${process.cwd()}/downloads`).calledOnce)
+    })
+    it('DOWNLOAD_DEFAULT_PATH is not empty', function () {
```

Comment:
same as above- the `it` should read like a sentence 😄

Reply:
ditto